easybuild/easyconfigs/h/HDF5/HDF5-1.10.5-intel-2019a.eb

```diff
@@ -0,0 +1,21 @@
+name = 'HDF5'
+version = '1.10.5'
+
+homepage = 'https://portal.hdfgroup.org/display/support'
+description = """HDF5 is a data model, library, and file format for storing and managing data.
+ It supports an unlimited variety of datatypes, and is designed for flexible
+ and efficient I/O and for high volume and complex data."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
```

Comment:
@akesandgren HDF5 doesn't use BLAS/LAPACK, so we can pull this down to the `iimpi` toolchain?

Reply:
Yeah should be doable.... will try

Reply:
Done